account/manager.go

```diff
@@ -153,13 +153,17 @@ func NewManager(cfg *ManagerConfig) *Manager {
 		quit: make(chan struct{}),
 	}
 
-	m.watcherCtrl = watcher.NewController(&watcher.Config{
-		ChainNotifier:       cfg.ChainNotifier,
+	eventHandler := watcher.NewEventHandler(&watcher.Handlers{
```

Comment:
I think we should just export those `handle*` methods on the manager and just pass `m` here instead of adding this wrapper struct.